### DISCUSSION TITLE
Versions for packages

### DISCUSSION THREAD

**Discussion by guyeatspants** [microsoft/vcpkg#23770] [Ideas]
I believe there should be a way to install a specific version of a package.   I haven't seen any documentation for that, I'm not sure if it's a thing yet.

Like if you make a vcpkg_rf.txt
`
install
openssl@1.1.1
smfl@2.5.2
`

For version 1.1.1 and 2.5.2 for example.

**Comment by Osyotr** [comment]
It is possible to install specific versions of packages when using manifest files:
https://github.com/microsoft/vcpkg/blob/master/docs/examples/versioning.getting-started.md
